sample/AppConfigDemo/packages.config

```diff
@@ -1,23 +1,21 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Data.SqlClient" version="1.1.3" targetFramework="net461" />
-  <package id="Microsoft.Data.SqlClient.SNI" version="1.1.0" targetFramework="net461" />
-  <package id="Microsoft.Extensions.Configuration.Abstractions" version="3.1.4" targetFramework="net461" />
-  <package id="Microsoft.Extensions.Primitives" version="3.1.4" targetFramework="net461" />
+  <!--<package id="Microsoft.Extensions.Configuration.Abstractions" version="3.1.4" targetFramework="net462" />-->
```

Comment:
Did you intentionally leave this commented out code?

Reply:
No, I forgot that. I will clean this up in the next release. Thank you! :)